src/components/sections/individual-adopters/Comments.tsx

```diff
@@ -0,0 +1,170 @@
+import React from "react";
+import styled from "styled-components";
+import { Row, SectionLight } from "../../layout";
+import { SubHeading } from "../../texts";
+import { adoptersContent } from "./data";
+
+interface Icontent {
+  url?: string;
+  adopter?: any;
+}
+
+const ContentSection = styled.div`
+  max-width: 1024px !important;
+  &:not(:first-child) {
+    margin-top: 3.75rem;
+  }
+`;
+
+const TextBox = styled.div`
+  margin-left: ${props => (props.theme.screens.md ? "1rem" : "2rem")};
+`;
+
+const List = styled.ul`
+  padding: 0 0 0 1.25rem;
+  margin: 1rem 0 0 0;
+`;
+
+const SubTextGray = styled.div`
+  color: ${props => props.theme.colors.darkGray};
+`;
+
+const Img = styled.img`
+  display: inline;
+  align-self: flex-start;
+  width: ${props => (props.theme.screens.md ? "3rem" : "")};
+`;
+
+const Why: React.FC<Icontent> = ({ adopter }) => {
+  return (
+    <ContentSection>
+      <Row>
+        <Img src={`/assets/bullets/${adopter.key}/why.svg`} />
+        <TextBox>
+          <SubHeading>{adopter?.why.heading}</SubHeading>
+          <br />
+          <SubTextGray>{adopter?.why.subtext}</SubTextGray>
+          <SubTextGray>
+            {adopter?.why.list ? (
+              <List>
+                {adopter?.why.list.map((adopter: any) => {
+                  return <li>{adopter}</li>;
+                })}
+              </List>
+            ) : (
+              <></>
```

Comment:
This can be remove as ternary is not required here

Reply:
Done!